src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md

```diff
@@ -53,15 +53,19 @@ We'll make several calls to find the values needed to create the product
 
 The sample data provides multiple attribute sets, including Default, Top, and Bottom. To assign the Top attribute set to the product, we need to know the corresponding `attribute_set_id`.
 
-Use the following call to search for the attribute set named `Top`.
+Use the following call to search for the attribute set named `Top` and entity_type_id equals to 4.
+By default, the `catalog_product` entity has the entity_type_id value as 4.
```

Comment:
```suggestion
By default, the `catalog_product` entity has the `entity_type_id` value as `4`.
```

Reply:
@BarnyShergold Thanks for the review. I have committed the changes.